modules/core/src/smithy4s/schema/Alt.scala

```diff
@@ -23,36 +23,54 @@ import kinds._
 /**
   * Represents a member of coproduct type (sealed trait)
   */
-final case class Alt[F[_], U, A](
+final case class Alt[U, A](
```

Comment:
parts of the changes in this PR is to eliminate the higher-kinded-ness of `Field` and `Alt`, as it's not been used at all ever since the initial-encoding of `Schema` became central to the codebase.